src/examples/table/TableDefault.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Row as GRow, Col } from '@zendeskgarden/react-grid';
+import { Body, Cell, Head, HeaderCell, HeaderRow, Row, Table } from '@zendeskgarden/react-tables';
+
+const Example = () => (
+  <GRow style={{ overflow: 'auto' }}>
+    <Col style={{ minWidth: '500px' }}>
```

Comment:
No need for a Grid here. This can probably just be `<div style={{ minWidth: 500 }}>`. If there are overflow scrolling issues, we should fix that at the `CodeExample` component level.

Reply:
...or maybe solve with `<div style={{ minWidth: 500, overflow: 'auto' }}>`. I haven't pulled to try locally.

Reply:
Nice that seems much more neat. How's the updated looking?

Reply:
Nice. Looks like we have a to-edge scrolling issue. But that's not your example problem. We need to update the `CodeExample` component to handle more gracefully.

[nit] I would use `style={{ minWidth: 500 }}`, but that's just me :wink: